src/lib/auth.ts

```
@@ -91,9 +92,9 @@ export async function registerUser(data: RegisterData) {
     });
 
     return response.register;
-  } catch (error) {
+  } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)